src/plugins/ejs-engine.ts

```diff
@@ -0,0 +1,35 @@
+import type {MojoContext} from '../types.js';
+import type {AsyncTemplateFunction} from 'ejs';
```

Comment:
Not alphabetical. And maybe add a newline between type imports and other imports.